jobs/JJOB_DA_PREP

```diff
@@ -98,12 +104,46 @@ ${ROOT_GODAS_DIR}/scripts/replace_KWRD_yaml.sh     \
       -k NO_ENS_MBR                                \
       -v ${NO_ENS_MBR}
 
+## Prep yaml for the soca_enspert for envar 
+#--------------------------------------------------------------
+${ROOT_GODAS_DIR}/scripts/prep_BKG_DATE_yaml.sh    \
+      -i $RUNCDATE/yaml/genenspert4envar.yml       \
+      -d ${bkg_date}
+
+${ROOT_GODAS_DIR}/scripts/replace_KWRD_yaml.sh     \
+      -i $RUNCDATE/yaml/genenspert4envar.yml       \
+      -k NO_ENS_MBR                                \
+      -v ${NO_ENS_MBR}
+
+${ROOT_GODAS_DIR}/scripts/replace_KWRD_yaml.sh     \
+      -i $RUNCDATE/yaml/genenspert4envar.yml       \
+      -k WINDOW_LENGTH                             \
+      -v ${window_length}
+
 # Prep yaml for 3dvar 
-#----------------------------------
+#-----------------------------
 ${ROOT_GODAS_DIR}/scripts/prep_3dvar_yaml.sh       \
-      -i $RUNCDATE/yaml/3dvar_godas.yml              \
+      -i $RUNCDATE/yaml/3dvar_godas.yml            \
       -d $RUNCDATE
 
+# Prep yaml for 3denvar 
+#----------------------------------
+${ROOT_GODAS_DIR}/scripts/prep_3denvar_yaml.sh    \
+      -i $RUNCDATE/yaml/3dhybenvar.yml            \
+      -m $RUNCDATE/yaml/envar_member.yml          \
+      -d $RUNCDATE                                \
+      -n ${NO_ENS_MBR}
```

Comment:
We use two different letters for the ${NO_ENS_MBR} -n and -v, let's keep only one.

Reply:
Ok, but:
-**i**nput
-**m**ember
-**d**ate
-**n**umber of member ... what does "v" stand for in the other script?